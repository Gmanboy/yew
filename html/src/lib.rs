use proc_macro_hack::proc_macro_hack;

/// Generate html tree
#[proc_macro_hack]
pub use yew_html_impl::html;
