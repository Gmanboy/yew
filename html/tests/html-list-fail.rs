use yew_html::html;

fn main() {
    html! {
        <><>
    };

    html! {
        </></>
    };

    html! {
        <><></>
    };

    html! {
        <></>
        <></>
    };
}
