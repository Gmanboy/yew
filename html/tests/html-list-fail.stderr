error: this open tag has no corresponding close tag
 --> $DIR/html-list-fail.rs:5:9
  |
5 |         <><>
  |         ^^

error: expected valid html element
 --> $DIR/html-list-fail.rs:9:9
  |
9 |         </></>
  |         ^

error: this open tag has no corresponding close tag
  --> $DIR/html-list-fail.rs:13:9
   |
13 |         <><></>
   |         ^^

error: only one root html element allowed
  --> $DIR/html-list-fail.rs:18:9
   |
18 |         <></>
   |         ^
